cmd/neofs-node/local_storage/config.go

```diff
@@ -0,0 +1,83 @@
+package local_storage
+
+import (
+	"io/fs"
+	"path/filepath"
+	"strings"
+	"time"
+
+	shardmode "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/shard/mode"
+)
+
+type ShardCfg struct {
```

Comment:
well, the new `internal` pkg? now `neofs-lens` imports `neofs-node`, not sure that is expected

Reply:
Why it can be unexpected? how to better name it? Or do you mean to move it to pkg somehow?